test/tests/serial/no_container.bats

```diff
@@ -8,20 +8,20 @@ load "${REPOSITORY_ROOT}/test/helper/common"
 BATS_TEST_NAME_PREFIX='[No Existing Container] '
 
 function setup_file() {
-  # Fail early if the testing image is already running:
-  assert_not_equal "$(docker ps | grep -o "${IMAGE_NAME}")" "${IMAGE_NAME}"
+  run docker ps # fail early if the testing image is already running:
+  assert_success
+  refute_output --partial "${IMAGE_NAME}"
 
-  # Copy the base config that `setup.sh` will volume mount to a container it runs:
   export TEST_TMP_CONFIG
   TEST_TMP_CONFIG=$(_duplicate_config_for_container . 'no_container')
 }
 
 @test "'setup.sh -p <PATH> -i <IMAGE>' should correctly use options" {
-  # Create a `postfix-virtual.cf` config to verify the container can access it:
+  # create a `postfix-virtual.cf` config to verify the container can access it:
```

Comment:
Our battle never ends with convention here :joy:

Reply:
Haha, probably! I will revert this though and keep your style.

```suggestion
  # Create a `postfix-virtual.cf` config to verify the container can access it:
```